scripts/test.sh

```diff
@@ -5,4 +5,4 @@ IFS=$'\n\t'
 cd "$( dirname "${BASH_SOURCE[0]}" )/.."
 
 >&2 echo "running gorson tests"
-go test -v ./...
+go test -v --count=1 ./...
```

Comment:
I am no sold on this change... just noticed that the tests were cached from my initial run even after i updated the packages. that just seemed weird to me.

Reply:
I think the problem might be the fact that we're using `--count` here instead of `-count`, which golang prefers for some reason.

Reply:
It didn't cache the run on my test, though.